### PR TITLE
feat: add infrastructure and orchestrator diagrams

### PR DESCRIPTION
**Pull request by mateuszjenek**

### CODE REVIEW

File: examples/dns/README.md [hunk 1]
```diff
@@ -1,8 +1,7 @@
 # Example: dns resource based on AWS Route 53
 
-This example configures a [dns](https://developer.humanitec.com/platform-orchestrator/reference/resource-types/#dns) Resource Definition using AWS Route 53.
-
-The created Resource Definition can be used in your Score file using:
+## Configuration
+This example configures a [dns](https://developer.humanitec.com/platform-orchestrator/reference/resource-types/#dns) Resource Definition using AWS Route 53. Workloads use the `dns` resource to dynamically create dns record would look like:
```
Comment (astromechza) [inline examples/dns/README.md]: nit: last sentence grammar is a bit off, perhaps "A workload using the `dns` resource to create dns records looks like:"?

File: examples/mysql/aurora/README.md [hunk 2]
```diff
@@ -1,8 +1,7 @@
 # Example: mysql resource based on AWS RDS Aurora
 
-This example configures a [mysql](https://developer.humanitec.com/platform-orchestrator/reference/resource-types/#mysql) Resource Definition using AWS RDS Aurora.
-
-The created definition can be used in your Score file using:
+## Configuration
+This example configures a [mysql](https://developer.humanitec.com/platform-orchestrator/reference/resource-types/#mysql) Resource Definition using AWS RDS Aurora. Workloads use the `mysql` resource to dynamically create databases would look like:
```
Comment (astromechza) [inline examples/mysql/aurora/README.md]: nit: same on the last sentence here

File: examples/mysql/basic/README.md [hunk 3]
```diff
@@ -1,8 +1,8 @@
 # Example: mysql resource based on AWS RDS
 
-This example configures a [mysql](https://developer.humanitec.com/platform-orchestrator/reference/resource-types/#mysql) Resource Definition using AWS RDS.
 
-The created Resource Definition can be used in your Score file using:
+## Configuration
+This example configures a [mysql](https://developer.humanitec.com/platform-orchestrator/reference/resource-types/#mysql) Resource Definition using AWS RDS. Workloads use the `mysql` resource to dynamically create databases would look like:
```
Comment (astromechza) [inline examples/mysql/basic/README.md]: nit: same on this one

File: examples/s3/README.md [hunk 4]
```diff
@@ -15,9 +16,32 @@ resources:
     class: basic-admin
 ```
 
+## Infrastructure setup
+The workload service account will automatically be assigned the necessary AWS IAM Role with the selected IAM Policy.
```
Comment (astromechza) [inline examples/s3/README.md]: nit: 
```suggestion
The workload service account will be automatically assigned to the necessary AWS IAM Role with the selected IAM Policy.
```

**Review by astromechza — COMMENTED** [review]
mostly nitpicks on some grammer repeated in a few places

File: examples/dns/README.md (hunk 1 above)
Comment (mateuszjenek) [reply examples/dns/README.md]: Thanks! Fixed

File: examples/mysql/aurora/README.md (hunk 2 above)
Comment (mateuszjenek) [reply examples/mysql/aurora/README.md]: Thanks! Fixed

File: examples/mysql/basic/README.md (hunk 3 above)
Comment (mateuszjenek) [reply examples/mysql/basic/README.md]: Thanks! Fixed

File: examples/s3/README.md (hunk 4 above)
Comment (mateuszjenek) [reply examples/s3/README.md]: Thanks! Fixed

File: examples/dns/README.md [hunk 5]
```diff
@@ -11,6 +10,30 @@ resources:
...
 
+## Infrastructure setup
+
+```mermaid
+graph TD;
+  subgraph AWS Hosted zone
+    record["record"]
+  end
+
+  subgraph AKS Cluster
```
Comment (johanneswuerbach) [inline examples/dns/README.md]: Nit: This should be `EKS`.

File: examples/postgres/aurora/README.md [hunk 6]
```diff
@@ -11,6 +10,30 @@ resources:
     type: postgres
 ```
 
+## Infrastructure setup
+
+```mermaid
+graph TD;
+  subgraph VPC
+    database["Postgres AWS RDS Aurora instance"]
+    sg["Postgres security group"]
```
Comment (johanneswuerbach) [inline examples/postgres/aurora/README.md]: Going through the security group looks a bit strange, maybe make the security group an annotation on the arrow?

File: examples/s3/README.md [hunk 7]
```diff
@@ -15,9 +16,32 @@ resources:
     class: basic-admin
 ```
 
+## Infrastructure setup
+The workload service account will be automatically assigned to the necessary AWS IAM Role with the selected IAM Policy.
+
+```mermaid
+graph TD;
+  s3["Amazon S3 bucket"]
+  policy["Amazon IAM Policy"]
```
Comment (johanneswuerbach) [inline examples/s3/README.md:25]: Should the policy also be pointed at `s3` to signal that policy grants access to this bucket?

**Review by johanneswuerbach — APPROVED** [review]
Left some minor comments

File: examples/s3/README.md (hunk 7 above)
Comment (mateuszjenek) [reply examples/s3/README.md:25]: Good idea! Added

File: examples/postgres/aurora/README.md (hunk 6 above)
Comment (mateuszjenek) [reply examples/postgres/aurora/README.md]: Hmm, you're right. Annotation looks better!

**Pull request merged**